src/OpenLoco/Windows/Vehicle.cpp

```diff
@@ -4051,6 +4051,16 @@ namespace OpenLoco::Ui::Windows::Vehicle
             Error::open(StringIds::cant_place_string_id_here, StringIds::null);
         }
 
+        static void pickupToolPlacementSuccess(Window& self, EntityId vehicleHead)
```

Comment:
In openrct2 we would call this a game command callback

Reply:
It's a lot of work to implement something like that here as the game commands are classes/objects in orct2, but in loco they're still just free functions. The callback is tied to the object there. Also, the only data passed to the callback is the command result, but in this instance we need the window context and the VehicleHead object, both of which aren't part of the result.

Obviously this could all be done still, but its a couple of full PRs work, I don't think it should be done here.

I did rename it though and rework slightly into more of a 'callback' style

Reply:
Yes just a rename is fine for now. At least then we have a common name for future.